sources/install.sh

```diff
@@ -1799,6 +1799,13 @@ function install_kerbrute() {
   chmod +x /opt/tools/bin/kerbrute
 }
 
+function install_hekatomb() {
+  colorecho "Installing Hekatomb"
+  git -C /opt/tools/ clone https://github.com/Processus-Thief/HEKATOMB
+  cd /opt/tools/HEKATOMB/
+  pip3 install -r requirements.txt
```

Comment:
Can you change this to `python3 -m pip install -r /opt/tools/HEKATOMB/requirements.txt` please, for uniformity?

Reply:
Resolved in second commit ;)